Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -1062,4 +1099,20 @@ -(UIView*)tryGetiOS16KeyboardFallbackViewFromParentView:(UIView*) parentView {
     return nil;
 }
 
+- (nullable UIView *)parentPlainScrollView
+{
+    UIView *currentSuperview = self.superview;
+    while (currentSuperview != nil) {
+        if ([currentSuperview isKindOfClass:[UIScrollView class]] &&
+            ![currentSuperview isKindOfClass:[UICollectionView class]] &&
```

Comment:
Probably want a comment in explaining why we don't want uicollectionviews and uitableviews